### PR TITLE
add highlights for 2022-07-08

### PR DESCRIPTION
**Pull request by weatherhog**
### Please remember to

- [x] Give the PR a meaningful title -- it will be shown in the release notes
- [ ] Add (or remove) [user questions](https://github.com/giantswarm/docs/blob/main/CONTRIBUTING.md#front-matter) associated with any updated docs

### CODE REVIEW

**Comment by gianfranco-l** [comment]
nothing from honeybadger for this week

File: src/content/changes/highlights/2022-07-08.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  - Highlights
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-product
+---
+
+## General
+
+## Apps
+
+## User interfaces
```
Comment (anvddriesch) [inline src/content/changes/highlights/2022-07-08.md:14]:
```suggestion
## User interfaces

## User interfaces

`kubectl gs login` now supports OIDC on workload clusters via `dex` and `athena`. Please check [the documentation on workload cluster OIDC configuration](https://docs.giantswarm.io/advanced/configure-dex-in-your-cluster/) as well as [the login command reference](https://docs.giantswarm.io/ui-api/kubectl-gs/login/) to find out more.
```

**Comment by stone-z** [comment]
Nothing from Shield this week

File: src/content/changes/highlights/2022-07-08.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+changes_categories:
+  - Highlights
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-product
+---
+
+## General
+
+## Apps
+
```
Comment (weatherhog) [inline src/content/changes/highlights/2022-07-08.md:13]:
```suggestion
- [cert-manager](https://github.com/giantswarm/cert-manager-app) version [v2.15.1](https://github.com/giantswarm/cert-manager-app/blob/master/CHANGELOG.md#2151---2022-07-07) Upgrades to upstream image [`v1.7.3`](https://github.com/jetstack/cert-manager/releases/tag/v1.7.3) which increases some hard-coded timeouts for certain ACME issuers (ZeroSSL and Sectigo) ([#243](https://github.com/giantswarm/cert-manager-app/pull/243)) and updates kubectl container version to `1.24.2` ([#243](https://github.com/giantswarm/cert-manager-app/pull/243))
```

**Review by anvddriesch — APPROVED** [review]

**Pull request merged**